deploy/olm-catalog/ibm-block-csi-operator/1.7.0/bundle-1.7.0.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM scratch
```

Comment:
Do we still need it?

Reply:
The keyword “scratch” is used to indicate that this layer is not built on top of anything else.
It's not mandatory, but I don't see a strong reason to remove it.